### ISSUE TITLE
Fix Your Timestep! link in README *on crates.io* is broken

### ISSUE THREAD

**Issue by dvberkel** [17cupsofcoffee/tetra#11]
Thanks for taking the time to create tetra. I saw your tweet on publishing it to crates.io and wanted to check it out.

Since you also wanted to get a heads up on any bugs I am creating this issue. Due note that it seems to me to be a issue with `crates.io` but I am not sure. I would like to investigate further, but in the meantime I would like to refer back to the first context I found it.

When I open [tetra](https://crates.io/crates/tetra) on crates.io and click on the Fix Your Timestep! link, I am redirected to a [GitHub 404 page](https://github.com/17cupsofcoffee/tetra/blob/master/*https://gafferongames.com/post/fix_your_timestep/).

The reason is that the url is

```plain
https://github.com/17cupsofcoffee/tetra/blob/master/*https://gafferongames.com/post/fix_your_timestep/
```

When I go to the [tetra repository](https://github.com/17cupsofcoffee/tetra) and click the same [Fix Your Timestep!](https://gafferongames.com/post/fix_your_timestep/) I end up at the correct article.

Like I said, this is in no part your problem. I will try to investigate what causes this, and then maybe transfer the link to crates.io

**Comment by 17cupsofcoffee** [comment]
Yep, I noticed this about 30 seconds after publishing, unfortunately 😅 

The issue is due to the fact I made a typo in my README before publishing the crate - I've fixed it on master already (which is why the link works on GitHub), but the change won't appear on crates.io until I publish a new version of the crate.

I'll probably be doing a 0.1.1 release in the near future, so it hopefully shouldn't be broken for too long! I'll leave this issue open till then. Thanks for reporting it 😀
